metadata-ingestion/src/datahub/ingestion/source/identity/azure_ad.py

```diff
@@ -299,17 +300,27 @@ def get_workunits(self) -> Iterable[MetadataWorkUnit]:
                         aspectName="origin",
                         aspect=OriginClass(OriginTypeClass.EXTERNAL, "AZURE_AD"),
                     )
-                    group_origin_wu_id = (
-                        f"group-origin-{group_count + 1}"
-                        if self.config.mask_group_id
-                        else datahub_corp_group_snapshot.urn
-                    )
+                    group_origin_wu_id = f"group-origin-{group_count + 1 if self.config.mask_group_id else datahub_corp_group_snapshot.urn}"
```

Comment:
interesting change. lint did this i guess

Reply:
Resolved offline! Wanted to make the WU ID more clear for unmasked cases